package.json

```diff
@@ -15,7 +15,16 @@
   "dependencies": {
     "commander": "2.9.0",
     "open": "0.0.5",
-    "shelljs": "0.5.3"
+    "shelljs": "0.5.3",
+    "react": "15.3.0",
+    "react-dom": "15.3.0",
+    "react-router": "2.0.1",
+
+    "redux": "3.3.1",
+    "react-redux": "4.4.1",
+    "react-router-redux": "4.0.0",
+    "reselect": "2.5.3",
+    "mout": "0.11.1"
   },
   "devDependencies": {
     "ws": "0.8.1",
```

Comment:
They are needed for visualizer: `ws`, `hammerjs`. 

Try to trash your node_modules and see if you can still build.